canvas_modules/common-canvas/package.json

```diff
@@ -26,84 +26,85 @@
     "redux": "4.0.5",
     "seedrandom": "3.0.5",
-    "uuid": "8.2.0"
+    "uuid": "8.3.0"
   },
   "devDependencies": {
-    "@babel/core": "7.10.4",
+    "@babel/core": "7.11.6",
     "@babel/plugin-proposal-class-properties": "7.10.4",
-    "@babel/plugin-transform-runtime": "7.10.5",
-    "@babel/polyfill": "7.10.4",
-    "@babel/preset-env": "7.10.4",
+    "@babel/plugin-transform-runtime": "7.11.5",
+    "@babel/polyfill": "7.11.5",
+    "@babel/preset-env": "7.11.5",
     "@babel/preset-react": "7.10.4",
-    "autoprefixer": "9.8.5",
+    "autoprefixer": "10.0.0",
     "babel-core": "6.26.3",
-    "babel-jest": "26.1.0",
+    "babel-jest": "26.3.0",
     "babel-loader": "8.1.0",
     "babel-plugin-istanbul": "6.0.0",
     "babel-plugin-lodash": "3.3.4",
     "chai": "4.2.0",
     "chai-enzyme": "1.0.0-beta.1",
-    "css-loader": "3.6.0",
+    "css-loader": "4.3.0",
     "deep-freeze": "0.0.1",
     "enzyme": "3.11.0",
-    "enzyme-adapter-react-16": "1.15.2",
-    "eslint": "7.4.0",
+    "enzyme-adapter-react-16": "1.15.5",
+    "eslint": "7.9.0",
     "eslint-config-canvas": "file:../eslint-config-canvas",
-    "eslint-plugin-react": "7.20.3",
-    "file-loader": "6.0.0",
-    "grunt": "1.2.1",
+    "eslint-plugin-react": "7.21.2",
+    "file-loader": "6.1.0",
+    "grunt": "1.3.0",
     "grunt-contrib-clean": "2.0.0",
     "grunt-env": "1.0.1",
     "grunt-eslint": "23.0.0",
-    "grunt-jsonlint": "2.1.2",
-    "grunt-webpack": "3.1.3",
+    "grunt-jsonlint": "2.1.3",
+    "grunt-webpack": "4.0.2",
     "grunt-yamllint": "0.3.0",
-    "jest": "26.1.0",
+    "jest": "26.4.2",
     "jest-fetch-mock": "3.0.3",
-    "jest-localstorage-mock": "2.4.2",
-    "mini-css-extract-plugin": "0.9.0",
+    "jest-localstorage-mock": "2.4.3",
+    "mini-css-extract-plugin": "0.11.2",
     "node-sass": "4.14.1",
-    "optimize-css-assets-webpack-plugin": "5.0.3",
+    "optimize-css-assets-webpack-plugin": "5.0.4",
     "path": "0.12.7",
-    "postcss-loader": "3.0.0",
-    "react": "16.12.0",
-    "react-dom": "16.12.0",
-    "react-intl": "5.0.2",
-    "react-redux": "7.2.0",
-    "react-test-renderer": "16.12.0",
+    "postcss": "8.0.9",
+    "postcss-loader": "4.0.2",
+    "react": "16.13.1",
+    "react-dom": "16.13.1",
+    "react-intl": "5.8.2",
+    "react-redux": "7.2.1",
+    "react-test-renderer": "16.13.1",
     "reactable": "1.1.0",
     "sass-lint": "1.13.1",
-    "sass-loader": "9.0.2",
-    "sinon": "9.0.2",
-    "source-map-loader": "1.0.1",
+    "sass-loader": "10.0.2",
+    "sinon": "9.0.3",
+    "source-map-loader": "1.1.0",
     "style-loader": "1.2.1",
-    "stylelint": "13.6.1",
+    "stylelint": "13.7.2",
     "url-loader": "4.1.0",
-    "webpack": "4.43.0",
-    "webpack-bundle-analyzer": "3.8.0"
+    "webpack": "4.44.2",
+    "webpack-bundle-analyzer": "3.9.0"
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
-    "react-intl": "^5.0.0",
-    "react-redux": "^7.0.0"
+    "react": "^16.13.1",
```

Comment:
These shouldn't have been updated.  We don't need the latest of these.  We are relying on the customer to install them.

Reply:
I'll revert these